packages/notifi-node/lib/types/WalletBlockchain.ts

```diff
@@ -1 +1,16 @@
-export type WalletBlockchain = 'SOLANA' | 'NEAR';
+export type WalletBlockchain =
+  | 'SOLANA'
+  | 'ETHEREUM'
+  | 'OFF_CHAIN'
+  | 'AVALANCHE'
+  | 'APTOS'
+  | 'ACALA'
+  | 'POLYGON'
+  | 'ARBITRUM'
+  | 'BINANCE'
+  | 'NEAR'
+  | 'OPTIMISM'
+  | 'INJECTIVE'
+  | 'SUI'
+  | 'OSMOSIS'
+  | 'EVMOS';
```

Comment:
It might be better not to include the unsupported chains:
- OFF_CHAIN
- EVMOS
- OSMOSIS